.github/workflows/sindri.yml

```
@@ -1,25 +1,97 @@
+        run: |
+          cd script
+          cargo build --release
+
+      - name: Test Zero-Knowledge Proof Generation
+        id: prove
+        run: |
+          cd script
+          # Generate proof with test values (no database required)
+          output=$(cargo run --release -- --prove --a 7 --b 13)
+          echo "$output"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)